content/docs/get-started/aws/deploy-stack.md

```diff
@@ -36,7 +36,7 @@ Do you want to perform this update?
   details
 ```
 
-Once the preview has finished you are given three options to choose from. Choosing `details` will show you a rich diff of the changes to be made. Choosing `yes` will create your new S3 bucket in AWS.
+Once the preview has finished, you are given three options to choose from. Choosing `details` will show you a rich diff of the changes to be made. Choosing `yes` will create your new S3 bucket in AWS. Choosing `no` will cancel the pending update operation.
```

Comment:
I think `cancel` isn't the right word here. When you choose `no` Pulumi doesn't cancel an `update` it just doesn't run the `update`.

Reply:
FWIW, I think this is okay wording, myself.

Reply:
@zchase I was referring to the user canceling/opt-ing out of going through with the update operation, not what's happening on the backend with Pulumi itself. I've updated the text to be more explicit about this.